src/commands/new-proj.ts

```diff
@@ -150,6 +180,37 @@ async function initDefaultProject() {
   }
 }
 
+/**
+ * Inits a React project
+ */
+async function initReactProject() {
+  try {
+    const cmd: string = await CreateReact(projName);
+    await runExecSync(cmd);
+
+    const cmds: string[] = await CleanReact(folderPath, repoPath);
+    for (let i = 0; i < cmds.length; i++) {
+      (i <= 8) ? await runExecSync(cmds[i]) : await runExecSync(cmds[i], { cwd: `.${path.sep}resources` });
```

Comment:
Are all commands really run in resources folder?

Reply:
Yes. See React Utils.